fe/fe-core/src/main/java/org/apache/doris/datasource/jdbc/client/JdbcClient.java

```diff
@@ -363,7 +363,7 @@ protected List<String> getSpecifiedDatabase(Connection conn) {
         } finally {
             close(conn);
         }
-        return databaseNames;
+        return jdbcLowerCaseMetaMatching.setDatabaseNameMapping(databaseNames);
```

Comment:
protected List<String> getSpecifiedDatabase(Connection conn) {
        List<String> remoteDatabaseNames = Lists.newArrayList();
        try {
            remoteDatabaseNames.add(conn.getSchema());
        } catch (SQLException e) {
            throw new JdbcClientException("failed to get specified database name from jdbc", e);
        } finally {
            close(conn);
        }
        return filterDatabaseNames(remoteDatabaseNames);
    }